src/main/java/Database/playlistAccessInterface.java

```diff
@@ -1,4 +1,34 @@
 package Database;
 
+import Entities.Playlist;
+
+import java.util.Collection;
+
 public interface playlistAccessInterface {
+
+    /**
+     * @return ArrayList of all existing Playlists
+     */
+    public Collection<Playlist> getPlaylists();
```

Comment:
I know we're supposed to try to be as abstract as possible in case we'll be using diff data types, but are you expecting on having different collections here? feel like the javadoc should match the actual type given here.

Reply:
That was a change I made to the code and missed in the javadoc, thanks for catching that. I’m returning a Collection since that’s how HashMaps work by default. Converting it to an ArrayList would require ugly casting or another loop, so I thought leaving it Collection would be fine since it’s also intervals. I’ll update the javadoc along with any other suggestions I get on this pull in next commit.